scripts/py_matter_idl/matter/idl/generators/cpp/sdk/Metadata.h.jinja

```diff
@@ -39,6 +39,14 @@ inline constexpr DataModel::AttributeEntry kMetadataEntry(
 } // namespace {{attribute.definition.name | upfirst}}
 {% endfor %}
 
+constexpr DataModel::AttributeEntry kMandatoryAttributes[] = {
```

Comment:
since this is in metadata and we name the metadata entries as `kMetdataEntry` would it make sense to name these `kMandatoryAttributesMetadata` (so that it does not look like a list of IDs for example)?